vite-project/src/strategypage/RadarChart.tsx

```diff
@@ -0,0 +1,266 @@
+import React, { useRef, useEffect } from "react";
+
+export interface point {
+  x: number;
+  y: number;
+}
+
+export interface RadarInput {
+  name: string;
+  value: number;
+  max: number;
+}
+
+export interface RadarComponentProps {
+  inputs: RadarInput[];
+}
+
+export const RadarComponent: React.FC<RadarComponentProps> = ({ inputs }) => {
+  const canvasRef = useRef<HTMLCanvasElement>(null);
+
+  const polygonPoints: point[] = new Array(inputs.length);
+  const statPoints: point[] = new Array(inputs.length);
+
+  //function that draws the outer regular polygos
+
```

Comment:
```suggestion
  // Draws (outer) ruler marks
```

Reply:
?

Reply:
Comment better
the next like already tells me directly that
- this is a `function`
- it `draw`s
- the `Outer`
- `Polygons`
if you're just gonna repeat that same information in your comment, why have a comment?